clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -845,6 +882,30 @@ private void handleListOffsetResponse(Map<TopicPartition, Long> timestampsToSear
         return parsedRecords;
     }
 
+    private PriorityQueue<FetchResponse.AbortedTransaction> getAbortedTransactions(FetchResponse.PartitionData partition) {
```

Comment:
Returning a `Queue` is fine, but not sure how to return an empty abstract queue. There isn't a `Collections.<>emptyQueue` or other such that I could find.